gigascience/gigadb-website
# Allows empty keywords in dataset entry

# Pull request for issue: #2061

This is a pull request for the following functionalities:

* In dataset form, e.g. http://gigadb.gigasciencejournal.com/adminDataset/update/id/5 allow to delete keywords by leaving field empty

## How to test?

navigate to any dataset, e.g. http://gigadb.gigasciencejournal.com/adminDataset/update/id/5, if there are no keywords, set a keyword and save, then remove keywords and save again. If you return to the dataset detail, no keywords should be present

## How have functionalities been implemented?

In the controller there is a conditional to update the keywords only if these are not empty, removed the condition so that if an empty field is received, keywords will be updated anyway

## Any issues with implementation?

--

## Any changes to automated tests?

--
